swagger/aggregator.yml

```diff
@@ -0,0 +1,81 @@
+openapi: 3.0.1
+info:
+  title: XAIN FL Aggregator API
+  contact:
+    email: engineering@xain.io
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.0
+externalDocs:
+  description: Find out more about XAIN FL
+  url: https://docs.xain.io/
+servers:
+  - url: http://localhost:8082/
+tags:
+  - name: Aggregator
+    description: API of the aggregator service
+paths:
+  /{client_id}/{client_token}:
+    get:
+      tags:
+        - Aggregator
+      parameters:
+        - name: client_id
+          in: path
+          description: ID of the client
+          required: true
+          example: 1fa2f908-83e2-4f92-80e1-4baee0bf25a5
+          schema:
+            type: string
+        - name: client_token
+          in: path
+          description: API token of the client
+          required: true
+          example: 2e8700a6-390d-411f-977b-9aa3bebc13e9
+          schema:
+            type: string
+      responses:
+        200:
+          description: weights of the global model
+          content:
+            application/octet-stream:
+              schema:
+                type: string
+                format: binary
+        404:
+          description: client unknown
+          content: {}
+    post:
+      tags:
+        - Aggregator
+      parameters:
+        - name: client_id
```

Comment:
`client_id` comes back in many requests. I think there's a way to define a `ClientId` type that would be re-used no? Maybe it's overkill here since we don't have that many routes though.